types/epoch.go

```diff
@@ -48,11 +47,24 @@ func (e *Epoch) String() string {
 		return e.name
 	}
 
-	return hexutil.EncodeBig(e.number)
+	return e.number.String()
+}
```

Comment:
Please help to add a util method as following:

`func (e *Epoch) ToInt() (*big.Int, bool)`